serum-390/soen390-team16-godzilla
# Production Frontend-to-Backend Connection

#168

Changes:
- Removed Materials table
- Populated Productions table with data
- Productions can be cancelled if they are not completed or have been cancelled yet
- New productions are made using a form
- Form requires an order ID and production date
- Also fixed the submitting 'null' bug when loading the purchases page, not yet for sales as that is still being worked on

Testing:
- Click "ADD NEW PRODUCTION"
- Type an order ID (try 1, 2, or 3)
- Choose a production date
- Press "Confirm"
- Cancel the production, or wait until it's completed
- Either way, the cancel button will be disabled for this production
